docs/src/pages/guides/styling.md

```diff
@@ -30,6 +30,32 @@ To create global styles, add a `:global()` wrapper around a selector (the same a
 <h1>I have both scoped and global styles</h1>
 ```
 
+To include every selector in a `<style>` as global styles, use `<style global>`. It's best to avoid using this escape hatch if possible, but it can be useful when importing global styling libraries like [Tailwind](/guides/styling#tailwind) or changing theme variables.
```

Comment:
Can you provide an example in the docs on how this helps with Tailwind? This is mentioned a couple of times but it's not clear to me on how it does, so users might also not understand, so an example would help.

Reply:
Yeah I think I'll actually just remove the mentions of how it helps with Tailwind.  My thought was that people might want to use this to wrap around the `@tailwind` setup, but that's not really the best example for when you'd want a global style block

Reply:
Just updated the docs to remove the Tailwind references, replaced it with a little more generic description of when you'd use it